include/sdf/Param.hh

```diff
@@ -430,6 +633,24 @@ namespace sdf
                 const std::optional<std::string> &_originalStr,
                 std::string &_valueStr) const;
 
+    /// \brief Method used to get the string representation from a ParamVariant,
+    /// or the string that was used to set it.
+    /// \param[in] _config Print configuration for the string output
+    /// \param[in] _typeName The data type of the value
+    /// \param[in] _value The value
+    /// \param[in] _orignalStr The original string that was used to set the
+    /// value. A nullopt can be passed in if it is not available.
+    /// \param[out] _valueStr The output string.
+    /// \param[out] _errors Vector of errors.
+    /// \return True if the string was successfully retrieved, false otherwise.
+    public: bool StringFromValueImpl(
```

Comment:
This is an internal function, so we can just change the original instead of creating an overload.

Reply:
Removing the overload and changing the original then.